Directory.Build.targets.example

```diff
@@ -1,21 +0,0 @@
-<Project>

```

Comment:
I see we have 2 Directory.Build.targets files now, one for each project. In Mapify we have just one, in the top folder of the repo. Can we do that here too?

Reply:
I don't honestly know enough about build.targets to answer that.